.eslintignore

```diff
@@ -3,4 +3,5 @@ node_modules/*
 vendor/*
 
 # Ignore build files
-.babelrc
\ No newline at end of file
+.babelrc
+dist/*
```

Comment:
This change doesn't seem entirely related to this PR. Not sure if that's a problem or not but maybe it should part of a PR that takes care of the updated build process. Not going to block it based on this but seems like something I should flag up.

Reply:
It's not related except that that was what was breaking linting.
![Screen Shot 2020-09-07 at 9 15 02 AM](https://user-images.githubusercontent.com/991511/92401311-a5080080-f0ea-11ea-8a09-8effd5e58432.png)